docs/contributing-devguide.md

```diff
@@ -124,4 +126,4 @@ Open `http://localhost:8000` in your browser.
 
 Since mike uses a branch to create/update documentation, any docs operation will create a diff on your local `gh-pages` branch.
 
-When finished writing/reviewing the docs, clean up your local docs branch changes with `git branch -D gh-pages`
\ No newline at end of file
+When finished writing/reviewing the docs, clean up your local docs branch changes with `git branch -D gh-pages`
```

Comment:
There was no newline here. Editing it in vim added it. If you want I can `:set nobinary` in vim to remove this.